.github/actions/prepare-env/action.yml

```diff
@@ -37,6 +37,7 @@ runs:
       run: |
         sudo apt update
         grep '^[^#]' apt-requirements.txt | xargs sudo apt install -y
+        sudo apt remove bison clang-format doxygen flex golang lld ninja-build -y
```

Comment:
```
The following packages were automatically installed and are no longer required:
  clang-format-14 golang-1.18 golang-1.18-doc golang-1.18-go golang-1.18-src
  golang-doc golang-go golang-src libclang-cpp14 libclang1-14 libedit2 libfl2
  libllvm14 libxapian30 lld-14
```
Further autoremove might be needed as this seems to only remove the meta package.